crates/uv/src/commands/pip_compile.rs

```diff
@@ -435,7 +444,14 @@ pub(crate) async fn pip_compile(
 
 /// Format the `uv` command used to generate the output file.
 #[allow(clippy::fn_params_excessive_bools)]
-fn cmd(include_index_url: bool, include_find_links: bool) -> String {
+fn cmd(
+    include_index_url: bool,
+    include_find_links: bool,
+    custom_compile_command: Option<String>,
+) -> String {
+    if let Some(cmd_str) = custom_compile_command {
+        return cmd_str;
+    }
```

Comment:
If above suggestion, revert
```suggestion
fn cmd(include_index_url: bool, include_find_links: bool) -> String {
```